### ISSUE TITLE
[FAST-57] [Firefox] At enable high contrast of windows accessibility page image not visible

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3073]
**Steps to Reproduce**
1) Go to the https://www.topcoder.com/community/accessibility-testing page.
2) Enable High contrast of windows
3) Got the new tab of browser and go to same url.
4) Observe whole the page background blank and image is not visible.

**Expected Result**
Expected to page reload proper visible form.

**Actual Result**
Background of accessibility-testing page gonna blank

**Device**: Desktop/Labtop
**Browser**: Firefox

**WCAG Category**: 1. Perceivable
**WCAG Level**: AAA
**WCAG Success Criterion**: 1.4.6 Contrast (Enhanced)
**Screen Reader Used**: 
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=2cd27847%2Dc944%2D4866%2Da87f%2D5eec4e505c52&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fkmurti%2FBug5

**Comment by kkartunov** [comment]
@Dara-K 
> tested the high contrast on my computer and I can see the banner well
![image (9)](https://user-images.githubusercontent.com/5585002/62270819-a541c100-b43f-11e9-8d47-1f91e9c7cecf.png)
